clang/lib/AST/Type.cpp

```diff
@@ -3907,6 +3923,31 @@ CountAttributedType::CountAttributedType(
     DeclSlot[i] = CoupledDecls[i];
 }
 
+StringRef CountAttributedType::getAttributeName(bool WithMacroPrefix) const {
+// TODO: This method isn't really ideal because it doesn't return the spelling
```

Comment:
It's not clear to me why this is needed at all -- doesn't `Attr::getSpelling()` suffice on the attribute, rather than asking the type?